apps/storybook/package.json

```diff
@@ -28,14 +28,10 @@
     "three": "0.172.0"
   },
   "devDependencies": {
-    "@esrf/eslint-config": "1.3.3",
-    "@storybook/addon-docs": "8.6.11",
-    "@storybook/addon-essentials": "8.6.11",
-    "@storybook/addon-links": "8.6.11",
-    "@storybook/addon-mdx-gfm": "8.6.11",
-    "@storybook/blocks": "8.6.11",
-    "@storybook/react": "8.6.11",
-    "@storybook/react-vite": "8.6.11",
```

Comment:
`@storybook/addon-mdx-gfm`, `@storybook/blocks` and `@storybook/react` no longer need to be installed.